server/index.js

```diff
@@ -10,9 +12,26 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '/../client/dist')));
 
+io.on('connection', socket => {
+  socket.on('orderSubmitted', payload => {
+    console.log('name and order received on server: ', payload);
+    io.emit('orderSubmitted', payload);
+  });
+
+  socket.on('onJoin', payload => {
+    console.log('Remote Name received on server: ', payload);
+    io.emit('onJoin', payload);
+  });
+});
+
 // the router for the users API
 app.use('/users', users);
 
-app.listen(PORT, () => {
-  console.log(`listening on port ${PORT}`);
+// OLD LISTEN
```

Comment:
Do we still need these commented out lines?

Reply:
Deleted!